client/src/constants/routes.js

```diff
@@ -1,7 +1,8 @@
+/* eslint-disable no-useless-escape */
 export default Object.freeze({
   // Hostname
-  ParticipantHostname: 'hcapparticipants.*',
-  EmployerHostname: 'hcapemployers.*',
+  ParticipantHostname: RegExp('^(www\.)?hcapparticipants\..*'),
+  EmployerHostname: RegExp('^(www\.)?hcapemployers\..*'),
```

Comment:
Because the linting rule is disabled, I'm not sure this will have its intended effect. Double backslash outta do it.
```suggestion
  EmployerHostname: RegExp('^(www\\.)?hcapemployers\\..*'),
```

Reply:
It has the correct effect, the linting rule doesn't change code functionality as far as I'm aware. Wouldn't escaping the backslash then match on hostnames that contained the literal backslash char, followed by any character? i.e. `www\Zhcapemployers.*`

Reply:
I believe `'^(www\\.)?hcapemployers\\..*'` evaluates to '^(www\.)?hcapemployers\..*' and seems to have the correct effect (doesn't allow `www*hcap`, just `www.hcap`) without the need to disable the lint rule, if there's a way to accomplish while making the linter happy we should go that direction